wcomponents-theme/src/test/intern/wc.file.util.test.js

```diff
@@ -0,0 +1,98 @@
+define(["intern!object", "intern/chai!assert", "./resources/test.utils!"],
+	function (registerSuite, assert, testutils) {
+		"use strict";
+		var fileUtil,
+			wcconfig;
+		registerSuite({
+			name: "wc/file/util",
+			setup: function () {
+				return testutils.setupHelper(["wc/file/util", "wc/config"], function (util, config) {
+					fileUtil = util;
+					wcconfig = config;
+				});
+			},
+			testBlobToFile: function () {
+				var expectedMime = "text/plain",
+					expectedFile = "test.file",
+					content = "This is my blob content",
+					blob = new Blob([content], {type: expectedMime}),
+					file = fileUtil.blobToFile(blob, {
+						name: expectedFile
+					});
+				assert.strictEqual(file.name, expectedFile);
+				assert.strictEqual(file.type, expectedMime);
+				assert.strictEqual(file.size, content.length);
+				// https://developer.mozilla.org/en-US/docs/Web/API/File#Properties
+				assert.isDefined(file.lastModified, "Check this is a file (not a blob).");
+				assert.strictEqual(file.lastModified, file.lastModifiedDate.getTime());
+				return new Promise(function (win) {
+					var fileReader = new FileReader();
+					fileReader.onload = function () {
+						assert.strictEqual(fileReader.result, content);
+						assert.notStrictEqual(fileReader.result, "content");
```

Comment:
If we already have done a strict equal check with `"This is my blob content"` does this second assertion test anything?

Reply:
I have added `notStrictEqual` based on our previous conversation that it is better to test success and failure to make sure the filereader has loaded the correct content (text)

Reply:
sure